src/packagedcode/models.py

```diff
@@ -117,14 +117,6 @@ class AssertedLicense(Model):
     notice = StringType(default=None)
     url = StringType(default=None)
 
-    def as_dict(self):
```

Comment:
ok, so do not remove `as_dict`. Make it available on the top level Package class and just have it call `to_primitive` : the point is that we do not want the external access to Packages to be tied to schematics.